domain_substitution.list

```diff
@@ -4772,7 +4771,6 @@ third_party/blink/renderer/core/dom/class_collection.cc
 third_party/blink/renderer/core/dom/class_collection.h
 third_party/blink/renderer/core/dom/comment.idl
 third_party/blink/renderer/core/dom/container_node.cc
-third_party/blink/renderer/core/dom/document.cc
```

Comment:
I am adding this because there are no remotely-accessed URLs to replace and it would break https://github.com/bromite/bromite/blob/78.0.3904.63/build/patches/Inject-scripts-for-AMP-tracking-ads-and-video-functionality.patch

Reply:
Sure. This also needs to be added to `devutils/update_lists.py`